### DISCUSSION TITLE
http2_max_concurrent_streams

### DISCUSSION THREAD

**Discussion by silence-coding** [hyperium/hyper#2868] [Q&A] [answered]
I do not want to run too many streams on one connection, because TCP and TLS generally have buffers for single connections.
As a server, I know that hyper has the http2_max_concurrent_streams configuration to control stream concurrency. But I see that hyper doesn't provide the configuration of the client(H2 crate has this configuration.).

**Comment by seanmonstar** [comment, marked as answer]
It would be fine to add the builder option to the client as well.
